### PR TITLE
Automatically remove "awaiting response" label on-comment

### PR DESCRIPTION
**Pull request by mitchhentges**
Also will close tickets that have been "awaiting response" for over 15 days.

### CODE REVIEW

**Comment by mitchhentges** [comment]
I have about 80% confidence that this is set up correctly:
* Is `${{ github.token }}` automatically filled in? [Sounds like it](https://github.com/lee-dohm/no-response#inputs), but unsure
* Will this behave properly since the `awaiting response` label already exists?
* `v0.5.0` :eyes: 
* I can't test auto-ticket closures in a fork

However, perusing through the active tickets in the `no-response` repo doesn't show anything too concerning, and the main benefit is that `awaiting response` labels will be auto-cleared on-response, which will make it easier to ensure that issues/PRs don't fall through the cracks, while we also don't waste time checking incomplete PRs/issues.

TL;DR: the failure case if this doesn't work is likely that this change will essentially be a no-op. I'm feeling lucky, though I trust your judgement Sylvestre.

**Comment by jschwe** [comment]
> This codecov report is reducing my trust in the tool, because I have no Rust code here.

I don't know too much about codecov, but we probably anyway want to switch to `-Zinstrument-coverage`, since that is on track to be [stabilized in 1.60](https://github.com/rust-lang/rust/pull/90132), while the `-Zprofile` option probably will never be stabilized. `instrument-coverage` should AFAIK be more accurate.

File: .github/workflows/awaiting-response.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    types: [created]
+  schedule:
+    - cron: '5 * * * *'
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          closeComment: >
```
Comment (ahochheiden) [inline .github/workflows/awaiting-response.yml:13]: The wording feels a bit weird. How about something more like this? (Feel free to put the "Of course," back in, I don't feel that strongly about it)
> This issue has been automatically closed because it has been blocked for too long on a response that wasn't provided. If the needed response is provided later, re-open this ticket so that we can progress it further. :)

**Review by ahochheiden — APPROVED** [review]
